.github/workflows/neon-preview-branches-for-pull-requests.yml

```diff
@@ -30,6 +30,15 @@ jobs:
     name: Create/Delete Branch for Pull Request Job
     runs-on: ubuntu-latest
     steps:
+      - name: Fetch integration data
+        shell: bash
+        run: |
+          WORKFLOW=$(curl -H "accept: application/json" -H "Authorization: Bearer ${{ secrets.NEON_API_KEY }}" https://console.neon.tech/api/v2/projects/${{ inputs.project_id }}/applications/github | jq -r ".workflows[] | select(.type == \"pr_branches\")")
+          echo $WORKFLOW
+          echo "workflow_parent=$(echo $WORKFLOW | jq .template_values.parent)" >> $GITHUB_ENV
+          echo "workflow_db=$(echo $WORKFLOW | jq .template_values.db)" >> $GITHUB_ENV
+          echo "workflow_role=$(echo $WORKFLOW | jq .template_values.role)" >> $GITHUB_ENV
+
       - name: Create Neon Branch
         uses: neondatabase/create-branch-action@v5.0.0
         if: |
```

Comment:
it makes sense to have the same `if` statement for the previous step. It makes no sense to call Neon API on other events.

Reply:
Fixed.